include/swift/AST/Attr.def

```diff
@@ -346,12 +346,6 @@ DECL_ATTR(_restatedObjCConformance, RestatedObjCConformance,
   OnProtocol |
   LongAttribute | RejectByParser |
   NotSerialized, 70)
-// HACK: Attribute needed to preserve source compatibility by downgrading errors
-// due to an SDK change in Dispatch
-SIMPLE_DECL_ATTR(_downgrade_exhaustivity_check, DowngradeExhaustivityCheck,
-  OnEnumElement |
-  LongAttribute | UserInaccessible,
-  71)
```

Comment:
Can you leave behind a "71 is unused comment" here, like we did for other enums?

Technically this should probably mean bumping the module version too